### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: 0.3.0

# Autonomy

- Adds `PYTHONHASHSEED` in Kubernetes deployment template
- Moves all fixtures into `aea_test_autonomy` plugin.
- Adds `service-registry-network` cli command for starting a local hardhat node. 

# Packages

- Replaces `third-party` dependencies with docker images. 
- Fixes import from tests folder and `path_to_skill` in FSM scaffolding
- Moves remaining tests into packages, in particular agents.

# Chores

- Adds README header
- Removes `quickstart` skip on command test
- Removes unnecessary shebangs from several non-script files
- Adds a script to validates commands in the docs and Makefile
- Cleans the `README.md` and `AUTHORS.md` to reflect changes

Describe in short the main changes with the new release.

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side), from `develop`
- [ ] I've updated the dependencies versions to the latest, wherever is possible.
- [ ] Lint and unit tests pass locally (please run tests also manually, not only with `tox`)
- [ ] I built the documentation and updated it with the latest changes
- [ ] I've added an item in `HISTORY.md` for this release
- [ ] I bumped the version number in the `__init__.py` file.
- [ ] I published the latest version on TestPyPI and checked that the following command work:
       ```pip install project-name==<version-number> --index-url https://test.pypi.org/simple --force --no-cache-dir --no-deps```
- [ ] After merging the PR, I'll publish the build also on PyPI. Then, I'll make sure the following
      command will work:
      ```pip install project-name-template==<version_number> --force --no-cache-dir --no-deps```  
- [ ] After merging the PR, I'll tag the repo with `v${VERSION_NUMVER}` (e.g. `v0.1.2`)

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -356,3 +357,31 @@ HelloWorldABCIAbstractRound
...
+Buitinck
+larsmans
+Thouis
+thouis
+scoder
+Phillip
+cpcloud
+Jevnik
+llllllllll
+3-Clause
```
Comment (DavidMinarsch) [inline .spelling:387]: ????

File: docs/upgrading.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ Below we describe the additional manual steps required to upgrade between differ
```
Comment (DavidMinarsch) [inline docs/upgrading.md:9]: Heading wrong!

File: plugins/aea-test-autonomy/setup.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def _get_docker_dependency() -> str:
 
 setup(
     name="open-aea-test-autonomy",
-    version="0.2.2",
+    version="0.3.0",
```
Comment (Karrenbelt) [inline plugins/aea-test-autonomy/setup.py:58]: `version=="3.0.0"`

File: docs/upgrading.md [hunk 4]
```diff
@@ -9,6 +9,13 @@ Below we describe the additional manual steps required to upgrade between differ
 
 No backwards incompatible changes except:
 
+- Deprecated the usage of `hashes.csv` and replaces it with `packages.json`, which is maintained by `autonomy packages lock`
+- `--check` flag is deprecated from `autonomy hash all`, from now package consistencies can be verified by `autonomy packages lock --check`
```
Comment (DavidMinarsch) [inline docs/upgrading.md:13]: - various test fixtures have moved and been renamed.

File: tests/test_base.py [hunk 5]
```diff
@@ -62,7 +62,7 @@ def get_test_files(package_type: PackageType) -> List[Path]:
 
 def test_version() -> None:
     """Test the version."""
-    assert autonomy.__version__ == "0.2.2"
+    assert autonomy.__version__ == "3.0.0"
```
Comment (DavidMinarsch) [inline tests/test_base.py:65]: wrong version

File: packages/valory/skills/liquidity_rebalancing_abci/skill.yaml [hunk 6]
```diff
@@ -130,5 +130,5 @@ models:
     class_name: SharedState
 dependencies:
   open-aea-test-autonomy:
-    version: ==0.2.2
+    version: ==3.0.0
```
Comment (DavidMinarsch) [inline packages/valory/skills/liquidity_rebalancing_abci/skill.yaml:133]: wrong version

File: autonomy/__version__.py [hunk 7]
```diff
@@ -22,7 +22,7 @@
 __title__ = "open-autonomy"
 __description__ = "A framework for the creation of autonomous agent services."
 __url__ = "https://github.com/valory-xyz/open-autonomy.git"
-__version__ = "0.2.2"
+__version__ = "3.0.0"
```
Comment (DavidMinarsch) [inline autonomy/__version__.py:25]: wrong version

File: .spelling (hunk 1 above)
Comment (angrybayblade) [reply .spelling:387]: Names from `AUTHORS.md`, I suggest we keep them temporary if this is an issue. We can deal with this later

Comment (DavidMinarsch) [reply .spelling:387]: Ugh this is not the right authors...you got them from some other repo! https://github.com/valory-xyz/open-autonomy/blob/main/AUTHORS.md

**Pull request merged**